pynucastro/templates/simple-cxx-network/screen.H.template

```diff
@@ -0,0 +1,218 @@
+#ifndef SCREEN_H
+#define SCREEN_H
+
+#include <cmath>
+#include <iostream>
+#include <string>
+
+#include <amrex_bridge.H>
+
+#include <network_properties.H>
+#include <fundamental_constants.H>
+#include <screen_data.H>
+
+struct plasma_state_t {
+    Real qlam0z;
```

Comment:
I think only `temp`, `n_e`, and `gamma_e_fac` are used for chugunov2007 though. Perhaps we should also group these variables based on which screening routine is used in Microphysics.

Reply:
good catch  -- I removed them